esp/services/ws_smc/ws_smcService.cpp

```diff
@@ -283,7 +283,7 @@ void CActivityInfo::readTargetClusterInfo(CConstWUClusterInfoArray& clusters, IP
     ForEachItemIn(c, clusters)
     {
         IConstWUClusterInfo &cluster = clusters.item(c);
-        if (cluster.isQueriesOnly()) //"publish-only" roxie queues not in SDS /JobQueues.
+        if (cluster.getRoxieTargetType() == RTTQueries) //"publish-only" roxie not in SDS /JobQueues.
```

Comment:
Could keep the helper function, and implement it as a comparison against the roxieTargetType.

Reply:
Will do